README.md

```diff
@@ -1,2 +1,20 @@
-# hello_world
-Project Hello World week 1
+Project Name
```

Comment:
- [ ] [OPTIONAL] This Pull Request should have a descriptive title telling you what changes are inside e.g: ``` Setup hello world project``` and the description (The first comment in the PR) should have a summary describing introduced changes in more detail, e.g: 
    ```
    - create HTML file
    - create CSS file
    - create test.md
    ...
    ```